cilium-cli/Makefile

```diff
@@ -1,28 +1,26 @@
 # Copyright Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-GO := go
-GO_BUILD = CGO_ENABLED=0 $(GO) build
-GO_TAGS ?=
-TARGET=cilium
+include ../Makefile.defs
+
+TARGET := cilium
+
+all: $(TARGET)
+
+.PHONY: all $(TARGET) local-release install clean test bench clean-tags tags
+
+
 INSTALL = $(QUIET)install
-BINDIR ?= /usr/local/bin
+BINDIR = /usr/local/bin
```

Comment:
FWIW this has already been fixed by https://github.com/cilium/cilium/pull/37911, a rebase would help.

Reply:
Thanks, that's a very relevant PR here indeed